src/System.IO.FileSystem.AccessControl/tests/System.IO.FileSystem.AccessControl.Tests.builds

```diff
@@ -5,9 +5,6 @@
     <Project Include="System.IO.FileSystem.AccessControl.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
-    <Project Include="System.IO.FileSystem.AccessControl.Tests.csproj">
```

Comment:
No, we need the source build to generate the dummy "platform not supported" assembly.